Formula/csound.rb

```diff
@@ -78,6 +85,12 @@ def install
   def caveats; <<~EOS
     To use the Python bindings, you may need to add to #{shell_profile}:
       export DYLD_FRAMEWORK_PATH="$DYLD_FRAMEWORK_PATH:#{opt_frameworks}"
+
```

Comment:
Is this homebrew specific?

Reply:
I can’t tell if this is referring to the caveat about Java bindings or the caveat about Python bindings, but the answer is yes either way.

The caveat about Java bindings is based on the caveats of the Subversion formula:

https://github.com/Homebrew/homebrew-core/blob/5df00040d5b7124764931a6edbd3e597e198db8c/Formula/subversion.rb#L145-L147

For Csound, setting the CLASSPATH environment variable (or, equivalently, using the `-classpath` option, as in the test) is also necessary. Without this, the test of Java bindings fails.

The reason for the caveat about Python bindings is described at https://github.com/Homebrew/homebrew-core/pull/39418#discussion_r286118804:

> This is needed because otherwise Python can’t find CsoundLib64.framework.
> 
> As far as I know, how dynamically-linked frameworks are found is described in the documentation of `dyld`. From `man dyld`:
> 
> > For each framework that a program uses, the dynamic linker looks for the framework in each directory in DYLD_FRAMEWORK_PATH in turn. If it looks in all the directories and can’t find the framework, it searches the directories in DYLD_LIBRARY_PATH in turn. If it still can’t find the framework, it then searches DYLD_FALLBACK_FRAMEWORK_PATH and DYLD_FALLBACK_LIBRARY_PATH in turn.
> 
> I believe DYLD_FRAMEWORK_PATH and DYLD_LIBRARY_PATH are both undefined (or empty) by default. The `dyld` man page says this about DYLD_FALLBACK_FRAMEWORK_PATH:
> 
> > **DYLD_FALLBACK_FRAMEWORK_PATH**
> >
> > This is a colon separated list of directories that contain frameworks. It is used as the default location for frameworks not found in their install path. By default, it is set to /Library/Frameworks:/Network/Library/Frameworks:/System/Library/Frameworks
> 
> The `dyld` man page says this about DYLD_FALLBACK_LIBRARY_PATH:
> 
> > **DYLD_FALLBACK_LIBRARY_PATH**
> >
> > This is a colon separated list of directories that contain libraries. It is used as the default location for libraries not found in their install path. By default, it is set to $(HOME)/lib:/usr/local/lib:/lib:/usr/lib.
> 
> Consequently, by default frameworks are searched for in these folders:
> 
> 1. /Library/Frameworks
> 2. /Network/Library/Frameworks
> 3. /System/Library/Frameworks
> 4. $(HOME)/lib
> 5. /usr/local/lib
> 6. /lib
> 7. /usr/lib
> 
> CsoundLib64.framework does get installed in /usr/local/Frameworks, but that’s not one of the default search paths.